forge-gui/res/cardsfolder/upcoming/racketeer_boss.txt

```diff
@@ -0,0 +1,14 @@
+Name:Racketeer Boss
+ManaCost:R G
+Types:Creature Cat Warrior
+PT:3/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChoose | TriggerDescription$ When CARDNAME enters the battlefield, you may choose 2 creature and/or planeswalker cards in your hand. They perpetually gain "When you cast this spell, create a Treasure token and this spell perpetually loses this ability."
+SVar:TrigChoose:DB$ ChooseCard | ChoiceZone$ Hand | Choices$ Creature.YouOwn,Planeswalker.YouOwn  | ChoiceTitle$ Choose up to two creature and/or planeswalker cards in your hand. | MinAmount$ 0 | Amount$ 2 | SubAbility$ DBEffect
```

Comment:
I think there's an extra space after Planeswalker.YouOwn

Maybe more the script over from upcoming to the r folder?

Other than that, looks good to merge